### PR TITLE
1339 add pm ratios to EMEP reader; add example config for them

### LINKED ISSUE #1339: add example aeroval configuration for PM10/PM25 ratio
**Describe a problem and/or suggest an improvement**
 there's no clear documentation of how to evaluate the PM10/PM25 ratio using aeroval

**Additional context**
add example config below `https://github.com/metno/pyaerocom/tree/main-dev/pyaerocom/aeroval/config`

### PR DESCRIPTION
**Pull request by jgriesfeller**
## Change Summary

This PR adds two more calculated variables to the EMEP reader:
`ratpm10pm25` and `ratpm25pm10`.
In addition, the aeroval config has been added which is available through 
`pyaerocom.aeroval.config.pmratios.base_config.get_CFG()`

## Related issue number

fix #1339 

## Checklist

* [x] Start with a draft-PR
* [x] The PR title is a good summary of the changes
* [x] PR is set to AeroTools and a tentative milestone
* [x] Documentation reflects the changes where applicable
* [x] Tests for the changes exist where applicable
* [x] Tests pass locally
* [x] Tests pass on CI
* [x] At least 1 reviewer is selected
* [x] Make PR ready to review

### CODE REVIEW

File: pyaerocom/io/mscw_ctm/additional_variables.py [hunk 1]
```diff
@@ -382,3 +382,47 @@ def calc_concpolyol(concspores):
     concpolyol = concspores.copy(deep=True) * factor
     concpolyol.attrs["units"] = "ug m-3"
     return concpolyol
+
+
+def calc_ratpm10pm25(concpm10: xr.DataArray, concpm25: xr.DataArray) -> xr.DataArray:
```
Comment (thorbjoernl) [inline pyaerocom/io/mscw_ctm/additional_variables.py:390]: I would like to see some more validation here, in particular an isinstance check on the inputs.

Should this validate that units are compatible?

File: pyaerocom/io/mscw_ctm/additional_variables.py [hunk 2]
```diff
@@ -382,3 +382,47 @@ def calc_concpolyol(concspores):
...
+
+    """
+    ratpm10pm25 = concpm10 / concpm25
+    ratpm10pm25.attrs["units"] = "1"
+    return ratpm10pm25
+
+
+def calc_ratpm25pm10(concpm25: xr.DataArray, concpm10: xr.DataArray) -> xr.DataArray:
+    """
+    Calculate ratio of pm10 and pm25
```
Comment (thorbjoernl) [inline pyaerocom/io/mscw_ctm/additional_variables.py:421]: Same as above

File: pyaerocom/io/mscw_ctm/reader.py [hunk 3]
```diff
@@ -702,7 +707,7 @@ def read_var(self, var_name, ts_type=None, **kwargs):
             proj_info=proj_info,
         )
 
-        #!obsolete
+        # !obsolete
```
Comment (thorbjoernl) [inline pyaerocom/io/mscw_ctm/reader.py]: Can this be removed?

File: tests/io/mscw_ctm/test_additional_variables.py [hunk 4]
```diff
@@ -80,6 +82,22 @@ def test_calc_concNtnh():
     assert (concNtnh == concNtnh_from_func).all()
 
 
+def test_calc_concpm10pm25():
+    concpm10 = create_fake_MSCWCtm_data()
+    concpm25 = create_fake_MSCWCtm_data()
+
+    ratpm10pm25_from_func = calc_ratpm10pm25(concpm10, concpm25)
+    assert ratpm10pm25_from_func.attrs["units"] == "1"
```
Comment (thorbjoernl) [inline tests/io/mscw_ctm/test_additional_variables.py:90]: Some more extensive tests would be nice, eg on the shape of the output array and maybe even the calculated values.

File: tests/io/mscw_ctm/test_additional_variables.py [hunk 5]
```diff
@@ -80,6 +82,22 @@ def test_calc_concNtnh():
...
+
+
+def test_calc_concpm25pm10():
+    concpm10 = create_fake_MSCWCtm_data()
+    concpm25 = create_fake_MSCWCtm_data()
+
+    ratpm25pm10_from_func = calc_ratpm25pm10(concpm25, concpm10)
+    assert ratpm25pm10_from_func.attrs["units"] == "1"
```
Comment (thorbjoernl) [inline tests/io/mscw_ctm/test_additional_variables.py:98]: Same as above

File: tests/io/mscw_ctm/test_aeroval_configs.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+from pyaerocom.aeroval.config.pmratios.base_config import get_CFG
+
+
+def test_ratpmconfig():
+    """short test if the example configuration for pm ratios is still in the code"""
+
+    reportyear = year = 2019
```
Comment (thorbjoernl) [inline tests/io/mscw_ctm/test_aeroval_configs.py:7]: Would be nice to include a test which runs the config with coarse data and checks that it works as expected.

**Review by thorbjoernl — CHANGES_REQUESTED** [review]

File: pyaerocom/io/mscw_ctm/additional_variables.py (hunk 1 above)
Comment (jgriesfeller) [reply pyaerocom/io/mscw_ctm/additional_variables.py:390]: no such tests in the current  about 20 methods in there. I added a unit sanity check anyhow, but the test data has no units so it's in a try block

File: pyaerocom/io/mscw_ctm/additional_variables.py (hunk 2 above)
Comment (jgriesfeller) [reply pyaerocom/io/mscw_ctm/additional_variables.py:421]: same as above

File: pyaerocom/io/mscw_ctm/reader.py (hunk 3 above)
Comment (jgriesfeller) [reply pyaerocom/io/mscw_ctm/reader.py]: yes. Done.

File: tests/io/mscw_ctm/test_additional_variables.py (hunk 5 above)
Comment (jgriesfeller) [reply tests/io/mscw_ctm/test_additional_variables.py:98]: test data has not the right unit. Doesn't make sense here.

File: tests/io/mscw_ctm/test_aeroval_configs.py (hunk 6 above)
Comment (jgriesfeller) [reply tests/io/mscw_ctm/test_aeroval_configs.py:7]: made this a separate issue #1379

**Review by thorbjoernl — APPROVED** [review]
LGTM

**Pull request merged**